### DISCUSSION TITLE
Why is `initialColorMode` forced to dark mode?

### DISCUSSION THREAD

**Discussion by RibhararnusPracutian** [chakra-ui/chakra-ui#5015] [Q&A]
I have set theme config below, but it seems the Chakra `v1.6.12` forcing color mode to system mode, is this expected?

```typescript
const config: ThemeConfig = {
  initialColorMode: "light",
  useSystemColorMode: false,
};
```

**Comment by daniel-schulz-codes** [comment]
Hej @RibhararnusPracutian thanks for reaching out. Unfortunatly there is a bug which was introduced in v1.6.12 that forces color mode to system mode https://github.com/chakra-ui/chakra-ui/issues/4987 we're currently working on this issue and hope to fix it asap.

**Reply by RibhararnusPracutian** [reply]
@takethefake wow okay thank you. 👍
